### ISSUE TITLE
Release updated Helm chart for v0.8.0

### ISSUE THREAD

**Issue by marquiz** [kubernetes-sigs/node-feature-discovery#501]
**What would you like to be added**:

Release an update on the Helm chart for v0.8.0. I'm still pondering how to do that, though. Obvious options that came to my mind:
1. Make a new point release v0.8.1 – seems like an overkill as there are no fixes in the app itself
2. Just update the existing artefact (`node-feature-discovery-chart-0.8.0.tgz`) – bad, release artefacts should be created from the tag
3. Create a new release artefact `node-feature-discovery-chart-0.8.0+1.tgz` in the v0.8.0 release page

To me, option 3. looks like the best one, haven't verified the semver magic, though

**Why is this needed**:

In the `release-0.8` there are bug fixes in the Helm chart branch that should be released (#496 still in the pipeline)

**Comment by marquiz** [comment]
Submitted #502 to help with 3. above

**Comment by CuBiC3D** [comment]
Please do no use obscure versioning that is not compatible with helm charts like `v0.8.0+1`. Helm requires a SemVer 2 (https://helm.sh/docs/topics/charts/) which does now allow a `v` prefix as part of the version itself (https://semver.org/) and especially states
```
Build metadata MAY be denoted by appending a plus sign and a series of dot separated identifiers immediately following the patch or pre-release version. [...] Build metadata MUST be ignored when determining version precedence. Thus two versions that differ only in the build metadata, have the same precedence.
```
Build metadata can not be used to mark a release that us following an other. The intended way is to decouple the chart versioning from the app versioning. I've too many projects doing this mistake in the past.

**Comment by marquiz** [comment]
> Please do no use obscure versioning that is not compatible with helm charts like `v0.8.0+1`. Helm requires a SemVer 2 (https://helm.sh/docs/topics/charts/) which does now allow a `v` prefix as part of the version itself (https://semver.org/) and especially states:

Of course not, the `v` prefix is only used in git and container image tags.

> Build metadata can not be used to mark a release that is following an other. The intended way is to decouple the chart versioning from the app versioning. I've seen too many projects doing this mistake in the past.

Hmm, I wasn't aware of this "feature", thanks for pointing this out. Looks like we go with option 1. then.

**Comment by marquiz** [comment]
Created #504

**Comment by marquiz** [comment]
Fixed by [v0.8.1](https://github.com/kubernetes-sigs/node-feature-discovery/releases/tag/v0.8.1)
